orangebeard.json

```diff
@@ -0,0 +1,16 @@
+{
+  "endpoint": "https://host.orangebeard.app",
+  "token": "722f2956-4ef9-455f-a8aa-2be5408ec6d0",
```

Comment:
I hope this is a UUID generated for the purpose, and not your personal one?

Reply:
yes, it's completely random :)